modules/schematics/src/action/index.ts

```diff
@@ -29,17 +30,19 @@ export default function(options: ActionOptions): Rule {
     options.path = parsedPath.path;
 
     const templateSource = apply(url('./files'), [
-      options.spec ? noop() : filter(path => !path.endsWith('__spec.ts')),
-      template({
+      options.spec
+        ? noop()
+        : filter(path => !path.endsWith('.spec.ts.template')),
+      applyTemplates({
+        ...stringUtils,
         'if-flat': (s: string) =>
           stringUtils.group(
             options.flat ? '' : s,
             options.group ? 'actions' : ''
           ),
-        ...stringUtils,
-        ...(options as object),
-        dot: () => '.',
-      } as any),
+        ...options,
+        dot: '.',
```

Comment:
Revert the dot changes or remove the `dot` altogether. They are no longer needed

Reply:
removed dot.